brvs-core/src/main/java/iroha/validation/Application.java

```diff
@@ -1,18 +1,38 @@
 package iroha.validation;
 
+import iroha.validation.rest.AccountValidityChecker;
 import iroha.validation.service.ValidationService;
-import org.springframework.context.annotation.ComponentScan;
+import iroha.validation.transactions.storage.TransactionVerdictStorage;
+import java.net.URI;
+import org.glassfish.hk2.utilities.binding.AbstractBinder;
+import org.glassfish.jersey.grizzly2.httpserver.GrizzlyHttpServerFactory;
+import org.glassfish.jersey.server.ResourceConfig;
+import org.springframework.context.support.AbstractApplicationContext;
 import org.springframework.context.support.FileSystemXmlApplicationContext;
 
-@ComponentScan("iroha.validation")
```

Comment:
Why do you remove all Spring annotations?

Reply:
they seem to be redundant in case of xml configuration